GrandCentralBoard/Widgets/WebsiteAnalytics/BlogPostsPopularityWidgetBuilder.swift

```diff
@@ -8,6 +8,26 @@
 import GrandCentralBoardCore
 
 
+private class BlogPostTitleTranslator: PathToTitleTranslating {
+    let pathPrefix: String?
```

Comment:
Yes, because I don't like polluting the namespace. And it is not useful anywhere outside this widget. And this widget builder already creates the needed instance.